python/test/t_Normal_std.py

```diff
@@ -19,6 +19,10 @@ def cleanPoint(inPoint):
             inPoint[i] = 0.0
     return inPoint
 
+# First dimension 1 with default ctor
```

Comment:
Please add three unit tests to cover the features:
* a multivariate normal with independent copula
* a multivariate normal with non diagonal covariance matrix
* a distribution with singularities

Reply:
@mbaudin47 No. 
```
/* Get the PDF singularities inside of the range - 1D only */
Point DistributionImplementation::getSingularities() const
{
  if (dimension_ != 1) throw NotDefinedException(HERE) << "Error: cannot ask for PDF singularities for multivariate distributions.";
  return Point(0);
}
```